atest/testdata/templates.py

```diff
@@ -501,4 +501,33 @@
   },
   "abc" : "This is a multline string\nwith\nhttp://www.google.de\na link inside",
   "Target//Name" : "gen3flex//dlt"
+}
+
+JSONFORMAT_NONE_TRUE_FALSE = {
+  "params": {
+    "glob": {
+      "gGlobalIntParam" : 1,
+      "gGlobalFloatParam" : 1.332,
+      "Null_variable": None,
+      "None_variable": None,
+      "True_variable": True,
+      "False_variable": False,
+      "gGlobalString"   : "This is a string",
+      "gGlobalStructure": {
+        "general": "general"
+      }
+    }
+  },
+  "preprocessor": {
```

Comment:
Hi Son,
preprocessor data and tests has to be removed as long as we don't provide a corresponding feature.
Thank you,
Thomas

Reply:
Hi Thomas,

This is just test data for JsonPreprocessor package selftest and as we define JsonPreprocessor and RobotFramework TestsuitesManagement are 2 independent packages. This test data is not related to the feature which providing in TestsuitesManagement, so I think we can keep this test data as it is for JsonPreprocessor selftest.

Thank you,
Son

Reply:
Hi Son,
accepted!
Thank you,
 Thomas